views/scss/qti/_choice.scss

```diff
@@ -40,6 +40,14 @@
         // ol
         .choice-area {
             margin-top: 25px;
+            .label-content {
+                display: inline-block;
+                .qti-block {
+                    img {
+                        width: 100%;
+                    }
+                }
+            }
```

Comment:
oops, I thought it is already working this way: so maybe the style I mistakenly integrated on Friday on your behalf should be removed here? https://github.com/oat-sa/extension-tao-itemqti/blob/master/views/scss/qti/_choice.scss
What do you think ?